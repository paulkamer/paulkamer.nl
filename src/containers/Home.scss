@import '../styles/variables';

.skills {
  &__skillslist {
    max-width: 35em;
    padding: 0;
    list-style: none;

    .skillsgroup {
      display: block;

      &__discipline {
        padding-right: 0.5em;
        font-weight: bold;
        line-height: 1.5em;
      }

      &__skillslist {
        display: inline;
        padding: 0;
      }

      &__skill {
        display: inline-block;

        &::after {
          padding: 0 0.5em;
          content: "\2022";
        }

        &:last-of-type::after {
          content: none;
        }
      }
    }
  }
}
