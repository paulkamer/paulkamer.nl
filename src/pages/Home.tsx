import React from "react";
import { Link } from "react-router-dom";

import SkillsList from "../components/SkillsList";
import JobsList from "../components/JobsList";

import { Discipline, skills } from "../data/skills";

const Home: React.FC = () => {
  const filteredSkills = skills.filter((skillsGroup) =>
    [Discipline.Frontend, Discipline.Backend, Discipline.Databases].includes(
      skillsGroup.discipline
    )
  );

  return (
    <article className="home-page">
      <section id="bio" className="bio">
        <h2>About me</h2>
        <p>
          I'm an experienced full-stack web developer, more inclined towards
          back-end development, from The Hague in The Netherlands.
        </p>
        <p>
          I've worked in the sports data industry for over 8 years. The last 4
          years I worked as a Tech Lead for{" "}
          <a href="https://start.me" target="_blank" rel="noopener noreferrer">
            start.me
          </a>
          , an online start page/bookmark manager service.
        </p>

        <p>
          My main interests & experience are: Ruby, Node, PHP, PostgreSQL, JS
          (vanilla, Vue & React), cloud & serverless.
        </p>
        <p>
          Download my CV: <a href="/docs/cv.pdf">Dutch</a> |{" "}
          <a href="/docs/cv_en.pdf">English</a>
        </p>
      </section>

      <section id="skills">
        <SkillsList skills={filteredSkills} />
        <p>
          For a complete overview, see the <Link to="/skills/">skills</Link>{" "}
          page.
        </p>
      </section>

      <section id="jobs">
        <JobsList />
      </section>

      <section className="projects">
        <h2>Side projects</h2>
        <p>
          For an impression of side projects I've worked on, see the{" "}
          <Link to="/projects/">Side projects</Link> page.
        </p>
      </section>
    </article>
  );
};

export default Home;
