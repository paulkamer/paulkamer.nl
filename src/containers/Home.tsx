import React from 'react';
import { Link } from 'react-router-dom';

import './Home.css';

const Home = () => {
  const skills = [
    'Ruby','PostgreSQL','Node.js','Vue.js','Nuxt.js','React','Redis','Git',
    'Elasticsearch','RabbitMQ','Resque','ArchiMate','UML','Docker','CI/CD',
    'Nginx','PHP','MySQL','Oracle'
  ]

  return (
    <article>
      <h1>Home</h1>
      Download my CV: <a href="/docs/cv.pdf">Dutch</a> | <a href="/docs/cv_en.pdf">English</a>

      <section id="skills">
        <h2>Skills</h2>
        <ul className="skillslist">
          {skills.map((value, index) => {
            return <li key={index}>{value}</li>
          })}
        </ul>
      </section>

      <section>
        <h2 id="jobs">Jobs</h2>

        <article>
          <h3>start.me</h3>
        </article>

        <article>
          <h3>Global Sports Media / Perform Group</h3>
        </article>

        <article>
          <h3>Sport-Track / MyLaps</h3>
        </article>
      </section>

      <section>
        <h2 id="projects">Projects</h2>
        See <Link to="projects">Projects page</Link>
      </section>
    </article>
  );
}

export default Home;
