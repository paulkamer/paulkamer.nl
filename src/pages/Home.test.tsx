import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import Home from './Home';

it('renders correctly without any trail items', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
