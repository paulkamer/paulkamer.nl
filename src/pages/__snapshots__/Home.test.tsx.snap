// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly without any trail items 1`] = `
<article
  className="home-page"
>
  <section
    className="bio"
    id="bio"
  >
    <h2>
      About me
    </h2>
    <p>
      I am a software architect with a background in full-stack development, specializing in C# .NET, Python, and TypeScript. I'm passionate about creating robust and scalable software solutions. I possess expertise in cloud technology, particularly Azure, as well as a strong focus on backend development.
    </p>
    <p>
      Throughout my career, I have actively mentored and provided guidance to fellow developers and acted as team lead for several projects. I am adept at analyzing business requirements and translating them into technical solutions, aligning with industry best practices and standards.
    </p>
    <p>
      Download my CV: 
      <a
        href="/docs/cv.pdf"
      >
        Dutch
      </a>
       |
       
      <a
        href="/docs/cv_en.pdf"
      >
        English
      </a>
    </p>
  </section>
  <section
    id="jobs"
  >
    <section
      className="jobs"
    >
      <h2>
        Jobs
      </h2>
      <article
        className="intro-article"
      >
        <h3
          className="intro-article__title"
        >
          <a
            className="intro-article__title-link"
            href="/jobs/netcompany"
            onClick={[Function]}
          >
            Netcompany
            <span
              className="intro-article__subtitle"
            >
               - May 2020 - current
            </span>
          </a>
        </h3>
        <section
          className="intro-article__container"
        >
          <div
            className="intro-article__image"
          >
            <a
              href="/jobs/netcompany"
              onClick={[Function]}
            >
              <img
                alt="Netcompany"
                src="/images/logo-netcompany.png"
              />
            </a>
          </div>
          <div
            className="intro-article__text"
          >
            <section>
              <p>
                Netcompany provides end to end services from advisory over development and implementation to operation of large IT solutions to large enterprises and governments. It is an international company with offices in 6 countries, with over 2.500 employees.
              </p>
              <p>
                My curreny role at Netcompany is Managing Architect.
              </p>
            </section>
          </div>
        </section>
      </article>
      <article
        className="intro-article"
      >
        <h3
          className="intro-article__title"
        >
          <a
            className="intro-article__title-link"
            href="/jobs/startme"
            onClick={[Function]}
          >
            start.me
            <span
              className="intro-article__subtitle"
            >
               - dec 2015 - April 2020
            </span>
          </a>
        </h3>
        <section
          className="intro-article__container"
        >
          <div
            className="intro-article__image"
          >
            <a
              href="/jobs/startme"
              onClick={[Function]}
            >
              <img
                alt="start.me"
                src="/images/logo-startme.png"
              />
            </a>
          </div>
          <div
            className="intro-article__text"
          >
            <section>
              <p>
                start.me is an online start page/bookmark manager for both personal and professional use.
              </p>
              <p>
                As a Senior Web Developer, and later Tech Lead, I've worked on numerous front-end, back-end and infrastructure-related projects.
              </p>
            </section>
          </div>
        </section>
      </article>
      <article
        className="intro-article"
      >
        <h3
          className="intro-article__title"
        >
          <a
            className="intro-article__title-link"
            href="/jobs/gsm_perform"
            onClick={[Function]}
          >
            Global Sports Media / Perform Group
            <span
              className="intro-article__subtitle"
            >
               - jan 2010 - nov 2015
            </span>
          </a>
        </h3>
        <section
          className="intro-article__container"
        >
          <div
            className="intro-article__image"
          >
            <a
              href="/jobs/gsm_perform"
              onClick={[Function]}
            >
              <img
                alt="Global Sports Media / Perform Group"
                src="/images/logo-perform.svg"
              />
            </a>
          </div>
          <div
            className="intro-article__text"
          >
            <section>
              <p>
                Global Sports Media (GSM) collects and syndicates sports statistics via a REST API, FTP push service and hosted (white-label) websites and widgets.
              </p>
              <p>
                After being fully acquired by Perform Group, I've worked as a Software Architect on, amongst other things, a project to introduce a new sports data collection platform.
              </p>
            </section>
          </div>
        </section>
      </article>
      <article
        className="intro-article"
      >
        <h3
          className="intro-article__title"
        >
          <a
            className="intro-article__title-link"
            href="https://www.mylaps.com"
            target="_blank"
          >
            Sport-Track / MyLaps
            <span
              className="intro-article__subtitle"
            >
               - may 2007 - dec 2009
            </span>
          </a>
        </h3>
        <section
          className="intro-article__container"
        >
          <div
            className="intro-article__image"
          >
            <a
              href="https://www.mylaps.com"
              target="_blank"
            >
              <img
                alt="Sport-Track / MyLaps"
                src="/images/logo-mylaps.svg"
              />
            </a>
          </div>
          <div
            className="intro-article__text"
          >
            <section>
              <p>
                Sport-Track developed a GPS-based live tracking and tracing system for sporting events.
              </p>
              <p>
                As a web developer I worked on a back-end portal written in PHP, which was backed by an Oracle database. Using this portal, organisers could manage their sporting events. It also served as a registration portal for participants.
              </p>
            </section>
          </div>
        </section>
      </article>
    </section>
  </section>
  <section
    id="certificates"
  >
    <section
      className="certificates"
    >
      <h2>
        Certificates
      </h2>
      <ul
        className="certificates__certificateslist"
      >
        <li
          className="certificate"
        >
          <a
            href="https://www.credly.com/badges/9ba87db8-82ce-4bf6-bc78-aef0e999e05f"
            target="_blank"
          >
            Azure Fundamentals
             (
            Apr 30, 2021
            )
          </a>
        </li>
        <li
          className="certificate"
        >
          <a
            href="https://learn.microsoft.com/en-us/users/paulkamer-0128/credentials/5818367a1b1c1d9"
            target="_blank"
          >
            Azure Developer Associate
             (
            May 31, 2021
            )
          </a>
        </li>
        <li
          className="certificate"
        >
          <a
            href="https://www.credly.com/badges/5393ae1c-dfe0-413a-beba-6d40c800f4c2"
            target="_blank"
          >
            Certified Kubernetes Application Developer (CKAD)
             (
            Nov 14, 2021
            )
          </a>
        </li>
        <li
          className="certificate"
        >
          <a
            href="https://learn.microsoft.com/en-us/users/paulkamer-0128/credentials/8b360250ee368418"
            target="_blank"
          >
            Azure Administrator Associate
             (
            Mar 3, 2023
            )
          </a>
        </li>
        <li
          className="certificate"
        >
          <a
            href="https://learn.microsoft.com/en-us/users/paulkamer-0128/credentials/e4cb17c3d5e0c7a8"
            target="_blank"
          >
            Azure Solutions Architect Expert
             (
            May 19, 2023
            )
          </a>
        </li>
        <li
          className="certificate"
        >
          <a
            href="https://www.credly.com/badges/ea9e2396-0f7f-4975-a95f-9b2e019eeaa1"
            target="_blank"
          >
            TOGAF 9 Certified
             (
            Feb 3, 2024
            )
          </a>
        </li>
      </ul>
    </section>
  </section>
</article>
`;
